english/eating-walls/marshmallows-for-breakfast/eat-breakfast.md

```diff
@@ -0,0 +1 @@
+Do you want to get breakfast[get breakfast](../breakfast/breakfast.md)
```

Comment:
It looks like this page is not reachable from the rest of the story. Make you sure you add a link to an existing page that leads to this page, or readers will not be able to find your addition.